packages/commonwealth/client/scripts/views/components/Comments/CommentEditor/CommentEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { CWText } from '../../component_kit/cw_text';
+import { User } from 'views/components/user/user';
+import Topic from 'client/scripts/models/Topic';
+import Account from 'client/scripts/models/Account';
+import app from 'state';
+import clsx from 'clsx';
+import { ReactQuillEditor } from '../../react_quill_editor';
+import { CWButton } from '../../component_kit/new_designs/cw_button';
+import { CWValidationText } from '../../component_kit/cw_validation_text';
+import BN from 'bn.js';
+import { weiToTokens, getDecimals } from 'helpers';
+import { ContentType } from 'types';
+import type { DeltaStatic } from 'quill';
+import './CommentEditor.scss';
+
+type CommentEditorProps = {
+  parentType: ContentType;
+  canComment: boolean;
+  handleSubmitComment: () => void;
+  errorMsg: string;
+  contentDelta: DeltaStatic;
+  setContentDelta: React.Dispatch<React.SetStateAction<DeltaStatic>>;
+  tokenPostingThreshold: BN;
+  activeTopic: Topic;
+  userBalance: BN;
+  disabled: boolean;
+  cancel: (e: any) => void;
+  isAdmin: boolean;
+  author: Account;
+  editorValue: string;
+  shouldFocus: boolean;
+};
+
+export const CommentEditor = ({
+  parentType,
+  canComment,
+  handleSubmitComment,
+  errorMsg,
+  contentDelta,
+  setContentDelta,
+  tokenPostingThreshold,
+  activeTopic,
+  userBalance,
+  disabled,
+  cancel,
+  isAdmin,
+  author,
+  editorValue,
+  shouldFocus,
+}: CommentEditorProps) => {
+  const decimals = getDecimals(app.chain);
+
+  return (
+    <div className="CommentEditor">
+      <div className="attribution-row">
+        <div className="attribution-left-content">
+          <CWText type="caption">
+            {parentType === ContentType.Comment ? 'Reply as' : 'Comment as'}
+          </CWText>
+          <CWText
+            type="caption"
+            fontWeight="medium"
+            className={clsx('user-link-text', { disabled: !canComment })}
```

Comment:
`clsx => getClasses`

Reply:
I'm not sure why that's preferable, tbh. Open to changing it, though.